manual/src/examples/nostr/channels.md

```diff
@@ -0,0 +1,62 @@
+# Channels Example
```

Comment:
This documentation belongs in the manual. Can you move it?

Reply:
don't get what you mean, it's in the manual already, no?